.github/workflows/scylla.yaml

```diff
@@ -0,0 +1,46 @@
+name: Build Scylla
+
+on:
+  schedule:
+    # only at 5AM Saturday
+    - cron: '0 5 * * SAT'
+
+permissions: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  BUILD_TYPE: RelWithDebInfo
+  BUILD_DIR: build
+
+jobs:
+  build-scylla:
+    runs-on: ubuntu-latest
+    container: fedora:40
+    steps:
+      - run: |
+          sudo dnf -y install git
+      - uses: actions/checkout@v4
+        with:
+          repository: scylladb/scylladb
+          submodules: true
+      - name: Update seastar submodule
+        run: |
+          git config --global --add safe.directory '*'
+          git fetch origin master
```

Comment:
don't we need to change directory into `seastar`?

Reply:
indeed!